games.html

```diff
@@ -14,13 +14,13 @@
         {% assign display = row.name | split: "_" | join: " " %}
     {% endif %}
     <div class="left">
-        <p>
-            <a href="{{w}}{{ row.name }}">{{ display }}</a>
+       
+            <a href="{{w}}{{ row.name }}"> <p>{{ display }}</p></a>
             <br>
             <a href="{{w}}{{ row.name }}">
-                <img src="/assets/images/games/fuchsia-{{ row.image }}.png" alt="{{ display }}" title="{{ display }}">
+                <img style="height: 360px;" src="/assets/images/games/fuchsia-{{ row.image }}.png" alt="{{ display }}" title="{{ display }}">
```

Comment:
Looks like all `games` logos are now 360x360 in size.
```suggestion
                <img src="/assets/images/games/fuchsia-{{ row.image }}.png" alt="{{ display }}" title="{{ display }}">
```